starter-core/src/test/groovy/io/micronaut/starter/feature/picocli/PicocliSpec.groovy

```diff
@@ -22,7 +22,7 @@ class PicocliSpec extends ApplicationContextSpec {
         String template = new BuildBuilder(beanContext, BuildTool.GRADLE)
                 .language(language)
                 .applicationType(ApplicationType.CLI)
-                .features(['kapt'])
+                .features(['ksp'])
```

Comment:
This is the default, if we're going to switch from using Kapt, we don't need to provide `ksp`, just remove this line